spec.md

```diff
@@ -22,11 +21,9 @@
   - [Broker Errors](#broker-errors)
   - [Orphans](#orphans)
 
-<div id="changes"/> 
-## Changes ##
+## <div id="changes"/> Changes
```

Comment:
my point was that I don't think you need to manually add the "div" here.   In fact, this this "div" in there it gets an anchor named "-changes" which might throw off tooling that assumes normal github md processing.